src/app/clark.component.scss

```diff
@@ -4,4 +4,9 @@ clark-cookies {
     bottom: 0;
     left: 0;
     right: 0;
+}
+
+.modal-container {
+    height: 475px;
+    width: 560px;
 }
```

Comment:
Will this work on mobile?

Reply:
@EVogel1999 this is not mobile friendly, will need to look into sendgrid docs to adjust their element sizing on our page.

Reply:
Nice context